charts/astronomer/values.yaml

```diff
@@ -5,7 +5,7 @@
 # This version number controls the default Airflow chart version that will be installed
 # when creating a new deployment in the system. This is also used to ensure all
 # child airflow deployments are kept up to date and on the latest version.
-airflowChartVersion: 0.20.0
+airflowChartVersion: 1.0.0-rc1-build10949
```

Comment:
This needs to be removed before merging.

Reply:
Trying to get this merged in today. Do you know what value I should bump this to? Does it come after this gets merged? https://github.com/astronomer/airflow-chart/pull/230

Reply:
Currently set to 0.20.1 https://github.com/astronomer/astronomer/blob/b181b8a8b2/charts/astronomer/values.yaml#L8

Reply:
yeah I think we need to bump it to get the changes we've being working on merged in. And I think that means bumping the version number to say 0.21.0

I'm not sure how to release a new version of the chart off of airflow-chart:main

Reply:
I think you should bump it to whatever "build" version gets built after @jedcunningham's PR merge on master of airflow-chart this morning.

Reply:
Ok I just made a prerelease of it here https://github.com/astronomer/airflow-chart/releases/tag/v0.21.0. Will populate this now.

Reply:
And once testing on dev and QA cluster is done. We can release 1.0.0 of airflow-chart and update astronomer/astronomer to include that.

Reply:
So I'll make a 1.0.0 pre-release of airflow chart now and populate that here. Once we are done with testing we can bump out of prerelease?

Reply:
but you will end up consuming the 1.0.0 tag right? Why cant we keep it as an `rc` version until we are done with testing?